### ISSUE TITLE
planned translations - tracking issue

### ISSUE THREAD

**Issue by rgommers** [numpy/numpy-surveys#1]
Let's keep this as a tracking issue for the translations that are planned / we have volunteers for. Comments welcome from anyone who wants to take the lead on an issue.

**Comment by jangid** [comment]
Thanks Ralf for creating this. I would be contributing to English -> Hindi translations.

**Comment by seberg** [comment]
I can/could do German, although I doubt it would be super useful. Not quite sure, but French might be an interesting target because it is probably a fairly common second language in Africa?

**Comment by rgommers** [comment]
Yeah, I wouldn't do German (or Dutch for that matter). Only the major languages where English would be a big barrier to participation. 99.x% of our German users will speak English well enough.

**Comment by jangid** [comment]
1. Should we start working on the `NumPy-survey-questionnaire.md`
document? Or we just wait for the finalized version.
2. What naming convention to use for filenames? Should I use
`NumPy-survey-questionnaire-hi.md` for Hindi translations?

**Comment by InessaPawson** [comment]
@jangid Thank you for volunteering to translate the NumPy community survey into Hindi, Pankaj! 
<br>1. We need to finalize the questionnaire first. I’ll contact you as soon as it’s ready.
<br>2. I’d prefer `NumPy-survey-questionnaire-hindi.md`

**Comment by jangid** [comment]
@InessaPawson 👍 Sounds like a plan. "-hindi.md" suffix is fine. More clear.

-- 
Pankaj Jangid

**Comment by InessaPawson** [comment]
@seberg Indeed all the Germans I've met so far are fluent in English. Thank you for offering your help anyways!

**Comment by pentschev** [comment]
I can help with Portuguese if necessary.

**Comment by InessaPawson** [comment]
@pentschev That would be great! I’ll contact you as soon as the questionnaire is ready for translations.

**Comment by zjpoh** [comment]
I just saw this. 😃 I can help with translating to Mandarin and/or Malay / Indonesian if that's in scope.

**Comment by rgommers** [comment]
Google tells me 23 million Indonesian speakers (sounds low?) and 39 million Malay speakers. I don't know where the cutoff is, but it could be a nice thing to do.

**Comment by zjpoh** [comment]
Ya. I think there are way more Indonesian speaker than Malaysian just by country population. But most Malaysian (can't speak for Indonesian) speaks English so I'm not sure if that's necessary.

**Comment by rgommers** [comment]
https://en.wikipedia.org/wiki/Languages_of_Indonesia says 80% of the 210 million Indonesians speak Indonesian. So then yes, seems very worthwhile to do.

**Comment by InessaPawson** [comment]
> I just saw this. 😃 I can help with translating to Mandarin and/or Malay / Indonesian if that's in scope.

Thank you, ZJ!

**Comment by kikocorreoso** [comment]
Hi all,

I can be in charge of the spanish translation. Just let me know when the questionare is finished.

**Comment by InessaPawson** [comment]
@kikocorreoso Wonderful!  It's great to have you on board!

**Comment by InessaPawson** [comment]
Yuki Dunn has volunteered to help us with the translation into Japanese. (She is not on GitHub. I'll make sure she receives the necessary info and docs to do the translations.)

**Comment by InessaPawson** [comment]
Hi, everyone! Thank you for volunteering to translate the inaugural NumPy community survey questionnaire. Our collaboration with the faculty and students from the Master’s program in Survey Methodology at the University of Michigan and the University of Maryland at College Park is underway. We should be ready to send you a document for translation in about a week. Please confirm your availability. 
If you have any questions don’t hesitate to leave a comment below.

**Comment by kikocorreoso** [comment]
I confirm my availability for s spanish translation.

Have a nice weekend.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by zjpoh** [comment]
Confirmed, thanks~

**Comment by kritisingh1** [comment]
Hello, I will be happy to volunteer for translating the questionnaire to Hindi. Confirming my availability as well!

**Comment by InessaPawson** [comment]
> Hello, I will be happy to volunteer for translating the questionnaire to Hindi. Confirming my availability as well!

Thank you, @kritisingh1!

**Comment by ivanov** [comment]
I'm happy to help with translating to Russian.

**Comment by InessaPawson** [comment]
Fantastic! Thank you, @ivanov!

**Comment by tkoyama010** [comment]
I can contribute for Japanese translation. How can I proceed with the translation process?

**Comment by InessaPawson** [comment]
> I can contribute for Japanese translation. How can I proceed with the translation process?

Thank you very much for offering your help, @tkoyama010! We have already secured a volunteer for the translation into Japanese. If anything changes, I’ll contact you first, and certainly keep your name in mind for future projects involving the knowledge of the Japanese language. 
In the meantime, we would truly appreciate if you spread the word about our survey, when it goes live, within your network. Please look for an announcement in the coming weeks!

**Comment by kikocorreoso** [comment]
Hi all,

Sorry for a late reply. I've been busy with other stuff.

Is it too late to help with the spanish translation?

**Comment by JoseGuzman** [comment]
I may also help with the Spanish translation! I'm a Spanish neuroscientist and data scientist (see guzman-lab.com) with experience in writing in English/Spanish.

**Comment by InessaPawson** [comment]
Thank you for getting in touch, Kiko @kikocorreoso! It's not too late. Please email me for further details.

**Comment by InessaPawson** [comment]
@JoseGuzman Thank you for offering your help, Jose! If Kiko is available to complete forward translations, we will need another volunteer for the second stage (Spanish into English).

**Comment by kikocorreoso** [comment]
Hi, @InessaPawson 
Yesterday I sent an email with details about the translation.

Please, let me know if everything is ok.

**Comment by InessaPawson** [comment]
@kikocorreoso Kiko, you’ve completed the entire translation and included a detailed commentary in less than 3 hours! I’m blown away. Thank you so much!

**Comment by PRUBHTEJ** [comment]
Greetings,
My name is Prubhtej Singh.
Hindi is my native language and I can surely help with Hindi translation.
Hoping to hear from you soon
Regards

**Comment by InessaPawson** [comment]
@PRUBHTEJ Thank you for reaching out, Prubhtej! We have already secured a volunteer for this task. I'll definitely keep you in mind for our future needs.

**Comment by PRUBHTEJ** [comment]
Thanks @InessaPawson for the consideration!!!
Regards

**Comment by mhmohona** [comment]
Hello! Is it too late to comment here? 😅  
I am available to translate in [Bangla](https://en.wikipedia.org/wiki/Bengali_language).

**Comment by InessaPawson** [comment]
@mhmohona That would be great, Mahfuza! Please email me for further details.

**Comment by Sylfrena** [comment]
Hi! @InessaPawson 

Are you still looking for a translator for Bangla to English? I am familiar with Bangla and would love to do this!

**Comment by InessaPawson** [comment]
@Sylfrena Wonderful! Please email me (inessa @ albuscode.org) for further details.
